src/components/Candidates.tsx

```diff
@@ -10,29 +10,27 @@ const Div = styled.div`
   font-size: 30px;
   float: right;
   padding-top: 50px;
-  padding-bottom: 50px;
-`;
 
-const Wrapper = styled.div`
-  width: 80%;
-  min-height: 100px;
-  display: flex;
-  flex-direction: row;
-  justify-content: space-around;
-`;
-
-const ImageDiv = styled.div`
-  height: 250px;
-  width: 100%;
-  border: 1px solid ${props => props.theme.blackColor}
-  margin-bottom: 20px;
+  .Wrapper{
+    padding-top: 50px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+  }
 `;
 
 const ImgBlock = styled.div`
   min-width: 250px;
   display: flex;
   flex-direction: column;
   align-items: center;
+
+  .ImageDiv{
```

Comment:
Try to use code convention per language - if JS is camelCase, then css is kebab-case